contracts/service-registry/src/state.rs

```diff
@@ -195,6 +214,64 @@ pub fn remove_service_override(
     Ok(())
 }
 
+pub fn update_verifier_authorization_status(
+    storage: &mut dyn Storage,
+    service_name: &ServiceName,
+    auth_state: &AuthorizationState,
+    verifiers: &[Addr],
+) -> error_stack::Result<(), ContractError> {
+    let mut authorized_count_change = 0i32;
```

Comment:
use `i16`, it will make it way easier to deal with and it's plenty for verifiers